main.go

```diff
@@ -413,11 +412,11 @@ func outputToJSON(issues chan *Issue) int {
 	return status
 }
 
-func runLinters(linters map[string]*Linter, paths, ellipsisPaths []string, concurrency int, exclude *regexp.Regexp, include *regexp.Regexp) (chan *Issue, chan error) {
```

Comment:
Yes, I think that would be good. The code in general could use some cleanup. It has "evolved" somewhat from its origins as a simple tool.